apps/src/applab/sanitizeHtml.js

```diff
@@ -74,6 +80,9 @@ function warnAboutUnsafeHtml(warn, unsafe, safe, warnings) {
           attribs: attribs
         };
       }
+    },
+    exclusiveFilter: function (element) {
+      return (ignoredTags.indexOf(element.tag) !== -1);
```

Comment:
[Documentation for `exclusiveFilter` option in the sanitize-html package](https://www.npmjs.com/package/sanitize-html#filters).